frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -525,37 +526,65 @@ final class BloopBspServices(
   }
 
   def run(params: bsp.RunParams): BspEndpointResponse[bsp.RunResult] = {
+    def getMainClass(project: Project, state: State): Either[Exception, bsp.ScalaMainClass] = {
+      params.dataKind match {
+        // TODO replace with RunParamsDataKind.ScalaMainClass once bsp updates
```

Comment:
Because of this, I propose we merge this right after the DAP implementation which upgrades the version. Otherwise we will most likely forget to fix this TODO :smile: